src/cryptonote_core/sent_transition/sent_transition.h

```diff
@@ -74,6 +71,8 @@ void transition(
         const transition_context& context,
         service_nodes::service_node_list::state_t& sns,
         cryptonote::BlockchainSQLite& sql,
-        network_type net);
+        network_type net,
+        service_nodes::block_add_result& add_result,
```

Comment:
Unless I missed something, the two places which call this function create a container of this type and pass it in, but then don't do anything with the container.